### PR TITLE
Update example page style & NFT

### PR DESCRIPTION
**Pull request by collinjackson**
- Updates to demo web app style
- Update NFT

https://linear.app/capsule-labs/issue/CPSL-814/demo-w-new-branding
Screenshot 2023-11-07 at 6 45 39 PM
Screenshot 2023-11-07 at 6 45 40 PM

### CODE REVIEW

File: src/nft-marketplace/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -6,26 +6,24 @@ import { Link } from "react-router-dom";
 
 const Menu = () => (
   <>
-    <Link target="_blank" rel="noreferrer" to='https://7f4shq8oyfd.typeform.com/to/F86oVLhb'><p>Join Beta</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://usecapsule.com'><p>Website</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://docs.usecapsule.com/'><p>View Docs</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://github.com/capsule-org/capsule-example'><p>View Demo App Github</p></Link>
+    <Link target="_blank" rel="noreferrer" to='https://github.com/capsule-org/capsule-example'><p>Demo</p></Link>
```
Comment (vr2amesh) [inline src/nft-marketplace/components/navbar/Navbar.jsx]: Any reason we are naming this "Demo"? It is routing to the Github code which some users may not have direct access to view

**Comment by vr2amesh** [comment]
Screenshot 2023-11-07 at 6 17 45 PM
 
nit: Can we center this text on the bottom?

File: src/nft-marketplace/components/navbar/Navbar.jsx (hunk 1 above)
Comment (collinjackson) [reply src/nft-marketplace/components/navbar/Navbar.jsx]: It's in the design. Not sure where it should link, @nityas do you have a preference?

**Comment by collinjackson** [comment]
> Screenshot 2023-11-07 at 6 17 45 PM
> nit: Can we center this text on the bottom?

Done

File: src/nft-marketplace/components/navbar/Navbar.jsx (hunk 1 above)
Comment (collinjackson) [reply src/nft-marketplace/components/navbar/Navbar.jsx]: Removed it for now

**Review by vr2amesh — APPROVED** [review]

**Pull request merged**